tests/test_hello_world.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from hello_app import app
+
+
+def test_hello_world_handler_correct_header_input():
+    """
+    test the hello_world handler to conform its 
+    returning the expected values based on specific input
+    """
+
+    # Create a test client
```

Comment:
can you remove all the one line the comments in both tests please, that was me explaining, we don't need them in the codebase.